src/napari_imagej/widgets/menu.py

```diff
@@ -62,15 +62,15 @@ def _showUI(self):
         # First time showing
         if not self.gui.isVisible():
             # First things first, show the GUI
-            ij().ui().showUI(self.gui)
+            ij().thread().queue(lambda: ij().ui().showUI(self.gui))
```

Comment:
Can we describe why we're using ij().thread().queue()? I didn't know that this function existed, so it'd probably be good for reminding myself why later. Just something like `# Use ij().thread().queue() to ensure GUI operations occur on the correct thread`

Reply:
Added in f2b90ae